deploy/helm/falcosidekick/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "2.11.1"
 description: A simple daemon to help you with falco's outputs
 icon: https://raw.githubusercontent.com/falcosecurity/falcosidekick/master/imgs/falcosidekick.png
 name: falcosidekick
-version: 0.1.11
+version: 0.1.12
 maintainers:
   - name: SweetOps
   - name: Issif
```

Comment:
@czunker please, add you as maintainer, you did a great job

Reply:
Done. Thank you.